pages/common/asar.md

```diff
@@ -0,0 +1,19 @@
+# asar
+
+> A file archiver for the Electron platform.
+
+- Archive a file or folder:
+
+`asar pack {{path/to/file}} {{archived.asar}}`
+
+- Extract an archive:
+
+`asar extract {{archived.asar}}`
+
+- Extract a specific file from an archive:
+
+`asar extract-file {{archived.asar}} {{file}}`
+
+- List files of an archive:
```

Comment:
Perhaps ...`inside an archive:` would read better?

Reply:
I agree. Or perhaps streamline it with other archivers, e.g. `7z` which uses „List the contents of an archive file”